facade/versions/elrondProxyFacadeV1_2.go

```diff
@@ -0,0 +1,15 @@
+package versions
+
+import (
+	"github.com/ElrondNetwork/elrond-proxy-go/facade"
+	"github.com/ElrondNetwork/elrond-proxy-go/process/v1_2"
+)
+
+type ElrondProxyFacadeV1_2 struct {
+	AccountsProcessor v1_2.AccountProcessorV1_2
+	*facade.ElrondProxyFacade
+}
+
+func (epf *ElrondProxyFacadeV1_2) GetShardIDForAddressV1_2(address string, additionalField int) (uint32, error) {
```

Comment:
this needs to become GetShardIDForAddress,
implement on the other 2 interfaces the same function (GetShardIDForAddress) and remove it from base

Reply:
I used a different name because it has an additional field and it is more clear this way

Reply:
ok